index.html

```diff
@@ -459,13 +459,6 @@ <h3>ecdsa-2019</h3>
 section also include the verification of such a data integrity proof.
         </p>
 
-        <p class="advisement">
-When the RDF Dataset Canonicalization Algorithm [[RDF-CANON]] is used,
-implementations of that algorithm MUST detect 
-<a data-cite="RDF-CANON#dataset-poisoning">dataset poisoning</a> 
-by default, and abort processing upon detection.
-        </p>
-
```

Comment:
<p class="advisement">
When the RDF Dataset Canonicalization Algorithm [[RDF-CANON]] is used,
implementations of that algorithm will detect 
<a data-cite="RDF-CANON#dataset-poisoning">dataset poisoning</a> 
by default, and abort processing upon detection.
        </p>